src/main/java/org/traccar/config/Keys.java

```diff
@@ -542,6 +542,13 @@ private Keys() {
             "database.saveEmpty",
             List.of(KeyType.CONFIG));
 
+    /**
+     * This property controls the maximum lifetime in miliseconds of a connection in the pool.
+     */
+    public static final ConfigKey<Integer> DATABASE_MAX_LIFETIME = new IntegerConfigKey(
```

Comment:
Can we just do what we do with the `DATABASE_MAX_POOL_SIZE`? Basically only set if the value is not zero.

Reply:
`DATABASE_MAX_LIFETIME` defines max lifetime for the connection. And I need to control this parameter on the app side.

Reply:
I don't think you understood my comment. I'm saying that we can avoid hardcoding the default value.

Reply:
done